app/src/main/res/xml/media_player_control_widget_info.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
+    android:configure="io.homeassistant.companion.android.widgets.media_player_controls.MediaPlayerControlsWidgetConfigureActivity"
+    android:initialKeyguardLayout="@layout/widget_media_controls"
+    android:initialLayout="@layout/widget_media_controls"
+    android:minWidth="250dp"
+    android:minHeight="40dp"
+    android:updatePeriodMillis="86400000"
+    android:widgetCategory="home_screen" />
+    <!--android:previewImage="@drawable/example_appwidget_preview"-->
```

Comment:
Ah, totally forgot to include that. I saw your other preview images, I'll try one like those.

Reply:
I added a preview image, but it's in light mode instead of dark mode.  Do you want it to be dark mode to match the rest or light mode to have an example of what the light mode widgets look like?